### PR TITLE
feat(webapp): allow non kyc

### PR DESCRIPTION
**Pull request by leisterfrancisco**
### What does this PR do?

- Allow non-kyc user to apply

### Steps to test

1. `make run`

### CODE REVIEW

**Comment by leisterfrancisco** [comment]
Final mail

image

File: hapi/src/routes/add-join-request.route.js [hunk 1]
```diff
@@ -14,9 +16,18 @@ module.exports = {
...
 
-      if (isAnInvitee || joinRequest?.length || !hasKYC)
+      if (isAnInvitee || joinRequest?.length)
         throw Boom.badRequest('Account does not meet requirements')
 
+      if (!hasKYC) {
+        mailUtil.send({
+          account: input.account,
+          to: input.email,
+          subject: 'Your Proton referral link is not yet active!',
```
Comment (xavier506) [inline hapi/src/routes/add-join-request.route.js:26]: I think we can improve this message by saying :

Further action is required to activate your proton affiliate account.

File: hapi/src/routes/send-confirmation.route.js [hunk 2]
```diff
@@ -15,10 +18,11 @@ module.exports = {
     try {
       for (const account of input.accounts) {
         const { email } = await findByAccount(account)
-        await sendConfirmation({
+        await send({
           account: account,
           to: email,
-          subject: 'You are ready to share your Proton referral link!'
+          subject: 'You are ready to share your Proton referral link!',
```
Comment (xavier506) [inline hapi/src/routes/send-confirmation.route.js:24]: Welcome to the proton affiliate program!

File: hapi/src/routes/send-join-request-rejection.route.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      for (const account of input.accounts) {
+        const hasKYC = await checkKyc(account)
+        const { email } = await findByAccount(account)
+
+        if (hasKYC || !email) continue
+
+        await send({
+          account: account,
+          to: email,
+          subject: 'Your Proton referral link is not yet active!',
```
Comment (xavier506) [inline hapi/src/routes/send-join-request-rejection.route.js:29]: Further action is required to activate your proton affiliate account.

File: hapi/src/utils/templates/mail.template.js [hunk 4]
```diff
@@ -74,6 +74,79 @@ const generateConfirmationMail = ({ account }) => {
...
+                  </div>
+              </tr>
+              <tr>
+                  <div style="background-color: #000; margin-left: auto; margin-right: auto; padding-top: 20px;">
+                      <p align="left" style="width: 348px; margin: 0px auto 8px auto; font-family: Arial; font-size: 12px; font-weight: normal; font-stretch: normal; font-style: normal; line-height: 1.33; letter-spacing: 1.5px; text-align: center; color: #fff;">
+                          THIS PROJECT WAS FUNDED THROUGH THE PROTON GOVERNANCE COMMITTEE WORKER PROPOSAL SYSTEM
+                      </p>
+                      <p align="left" style="flex-grow: 0; margin: 0px 16px 14px; font-family: Arial; font-size: 14px; font-weight: normal; font-stretch: normal; font-style: normal; line-height: 1.14; letter-spacing: 0.44px; text-align: center; color: #fff;">
+                          <a href="https://forms.gle/GWHig5ciAvg5fdEH7" style="color: #fff;">
+                              Apply Here for Funding
```
Comment (xavier506) [inline hapi/src/utils/templates/mail.template.js:119]: does this need to be in the email?

**Review by tetogomez — APPROVED** [review]
LGTM

**Comment by leisterfrancisco** [comment]
After comments

image

**Review by xavier506 — APPROVED** [review]

**Pull request merged**